### PR TITLE
INCR-508 - Run python-modernize on ./ecommerce/extensions/checkout

### PR DESCRIPTION
**Pull request by ihtram**

### CODE REVIEW

File: ecommerce/extensions/checkout/mixins.py [hunk 1]
```diff
@@ -20,6 +21,7 @@
 from ecommerce.extensions.offer.constants import OFFER_ASSIGNED, OFFER_ASSIGNMENT_REVOKED, OFFER_REDEEMED
 from ecommerce.extensions.order.constants import PaymentEventTypeName
 from ecommerce.invoice.models import Invoice
+from six.moves import range  # pylint: disable=import-error,ungrouped-imports
```
Comment (crice100) [inline ecommerce/extensions/checkout/mixins.py]: Could we group the imports instead of disabling the _ungrouped-imports_ check?

Comment (ihtram) [reply ecommerce/extensions/checkout/mixins.py]: @crice100 In that case we have to skip isort check on this group. Let me commit this change.

Comment (ihtram) [reply ecommerce/extensions/checkout/mixins.py]: @crice100 I have updated the code. Please review again

File: ecommerce/extensions/checkout/mixins.py [hunk 2]
```diff
@@ -21,6 +21,10 @@
 from ecommerce.extensions.order.constants import PaymentEventTypeName
 from ecommerce.invoice.models import Invoice
 
+import six  # isort:skip
```
Comment (crice100) [inline ecommerce/extensions/checkout/mixins.py]: @cpennington Could you look at this to see if ecommerce's isort or pylint configs should be updated? It seems odd to me that we need to skip sorting to have a clean build.

Comment (cpennington) [reply ecommerce/extensions/checkout/mixins.py]: After rebasing onto master w/ the latest isort, I think this is fixed.

**Comment by cpennington** [comment]
Unfortunately, even upgrading both isort and pylint doesn't fix this: See https://github.com/PyCQA/pylint/pull/2824, which has been merged to pylint master, but not yet released.

I'm going to continue w/ the pylint upgrade anyway, though.

**Review by crice100 — APPROVED** [review]

**Pull request merged**